### PR TITLE
added a missing '{' to fix my comet animation

### PR DESCRIPTION
**Pull request by terrifricker**
My animation wasn't  working anymore.

### CODE REVIEW

**Comment by DavidMatthewFraser** [comment]
make sure you are only making a pull request when you are ready for your changes to reviewed and merged 😊

**Comment by terrifricker** [comment]
Yep, this is a real pull request.

**Comment by DavidMatthewFraser** [comment]
I'm sorry for the confusion, I thought this was an extension of your last PR. That's what I get for skimming 😂.

File: assets/css/challenge2.css [hunk 1]
```diff
@@ -246,6 +246,7 @@ width:10%;
    top:40%;
    left:2000px;
   }
+}
```
Comment (DavidMatthewFraser) [inline assets/css/challenge2.css:249]: good catch

**Pull request merged**